Source/WebKit/InspectorGResources.cmake

```diff
@@ -8,11 +8,10 @@ macro(WEBKIT_BUILD_INSPECTOR_GRESOURCES _derived_sources_dir)
         VERBATIM
     )
 
-    add_custom_command(
-        OUTPUT ${_derived_sources_dir}/InspectorGResourceBundle.c ${_derived_sources_dir}/InspectorGResourceBundle.deps
-        DEPENDS ${_derived_sources_dir}/InspectorGResourceBundle.xml
-        DEPFILE ${_derived_sources_dir}/InspectorGResourceBundle.deps
-        COMMAND glib-compile-resources --generate --sourcedir=${_derived_sources_dir}/InspectorResources/WebInspectorUI --target=${_derived_sources_dir}/InspectorGResourceBundle.c --dependency-file=${_derived_sources_dir}/InspectorGResourceBundle.deps ${_derived_sources_dir}/InspectorGResourceBundle.xml
-        VERBATIM
+    GLIB_COMPILE_RESOURCES(
+        OUTPUT        ${_derived_sources_dir}/InspectorGResourceBundle.c
+        DEPFILE       ${_derived_sources_dir}/InspectorGResourceBundle.deps
```

Comment:
I think we don't need to specify `DEPFILE` on each call to the `GLIB_COMPILE_RESOURCES()` function, please read on below for an alternative.